src/Datadog.Trace.ClrProfiler.Managed/AutoInstrumentation/WinHttpHandler/WinHttpHandlerIntegration.cs

```diff
@@ -25,6 +25,7 @@ public class WinHttpHandlerIntegration
         private const string IntegrationName = nameof(IntegrationIds.HttpMessageHandler);
         private static readonly IntegrationInfo IntegrationId = IntegrationRegistry.GetIntegrationInfo(IntegrationName);
         private static readonly IntegrationInfo WinHttpHandlerIntegrationId = IntegrationRegistry.GetIntegrationInfo(nameof(IntegrationIds.WinHttpHandler));
+        private static readonly Vendors.Serilog.ILogger Log = Datadog.Trace.Logging.DatadogLogging.GetLogger(typeof(WinHttpHandlerIntegration));
```

Comment:
nit: I think we can remove this instance if we are not going to use it.

Reply:
Fixed in https://github.com/DataDog/dd-trace-dotnet/pull/1170/commits/5a3a3aa6ec6cc39e36ecd61366207fda020f57be